### PR TITLE
Added jquery-ui-timepicker-addon@1.6.1 with auto npm update

### PR DESCRIPTION
**Pull request by ecabuk**

### CODE REVIEW

**Comment by PeterDaveHello** [comment]
@cdnjs/intern please help me review this PR, thanks.

**Comment by maruilian11** [comment]
hi, @ecabuk 
i'm here to help you send the PR , you seem to add `jquery-ui-timepicker*.js` twice, please remove those files under 1.6.1, and keep files under 1.6.1/i18n.
and please change npmfilemap `**/*.+(js|css)` to avoid adding `index.html`.

ps. you can use `git rebase` to resend **a new and clear commit** to avoid reopening a new PR.
thank you for your contribution!

**Comment by ecabuk** [comment]
@maruilian11 It looks like cdnjs-importer needs some improvements, fixed, thanks!

**Comment by maruilian11** [comment]
@PeterDaveHello can we use `-r !(index.html|jquery-ui-sliderAccess.js)` as npmfliemap to avoid adding index.html and jquery-ui-sliderAccess.js but adding other files and files under **folder**.

**Comment by PeterDaveHello** [comment]
Why to make it complex if there is a easy way?

**Comment by PeterDaveHello** [comment]
Missing minified `jquery-ui-sliderAccess.js`

File: ajax/libs/jquery-ui-timepicker-addon/package.json [hunk 1]
```diff
@@ -1,17 +1,28 @@
...
     "time",
+    "timepicker",
+    "jquery",
+    "datetime",
     "jquery-ui"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/trentrichardson/jQuery-Timepicker-Addon"
+    "url": "https://github.com/trentrichardson/jQuery-Timepicker-Addon.git"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-ui-timepicker-addon/package.json]: unrelated change

File: ajax/libs/jquery-ui-timepicker-addon/package.json [hunk 2]
```diff
@@ -1,17 +1,28 @@
 {
   "name": "jquery-ui-timepicker-addon",
-  "filename": "jquery-ui-timepicker-addon.min.js",
-  "version": "1.4.5",
-  "description": "The timepicker addon adds a timepicker to jQuery UI Datepicker, thus the datepicker and slider components (jQueryUI) are required for using any of these. In addition all datepicker options are still available through the timepicker addon.",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-ui-timepicker-addon/package.json]: Why to remove this here?

File: ajax/libs/jquery-ui-timepicker-addon/package.json [hunk 3]
```diff
@@ -1,17 +1,28 @@
...
+  "npmName": "jquery-ui-timepicker-addon",
+  "npmFileMap": [
+    {
+      "basePath": "/dist",
+      "files": [
+        "**/*.+(js|css)"
+      ]
+    }
+  ],
+  "filename": "jquery-ui-timepicker-addon.min.js"
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-ui-timepicker-addon/package.json]: No need to touch filename.

File: ajax/libs/jquery-ui-timepicker-addon/package.json [hunk 4]
```diff
@@ -1,17 +1,28 @@
...
     "jquery-ui"
   ],
   "repository": {
     "type": "git",
-    "url": "https://github.com/trentrichardson/jQuery-Timepicker-Addon"
+    "url": "https://github.com/trentrichardson/jQuery-Timepicker-Addon.git"
   },
-  "license": "MIT"
+  "license": "MIT",
+  "homepage": "http://trentrichardson.com/examples/timepicker",
```
Comment (PeterDaveHello) [inline ajax/libs/jquery-ui-timepicker-addon/package.json]: no need to touch homepage

**Comment by ecabuk** [comment]
Even if I add the minified version of that file for this version, when system automatically update it to new version it'll be missing again.

It's not a hard dependency for this package, I can exclude it on npmFileMap option?

**Comment by PeterDaveHello** [comment]
@ecabuk yep but I'll handle it, not only this lib has this issue.

**Comment by ecabuk** [comment]
Done...

**Comment by PeterDaveHello** [comment]
Okay @ecabuk , thanks, because the parent of this PR's commit was 2 weeks ago, please rebase on our latest master branch to update it, so that I can merge it, thanks.

**Comment by ecabuk** [comment]
Done, thanks!

**Comment by PeterDaveHello** [comment]
Ooops ...

Duplicated files:
- ajax/libs/jquery-ui-timepicker-addon/1.6.1/jquery-ui-timepicker-addon-i18n.js
- ajax/libs/jquery-ui-timepicker-addon/1.6.1/jquery-ui-timepicker-addon-i18n.min.js

Already been located at i18n folder.

So please remove them, thanks.

**Comment by ecabuk** [comment]
C'mon! If you don't want to merge it just say it! Just kidding :) 

Separated files under the i18n folder but that is the packaged version that contains all. Maybe someone want to load them all. Doesn't matter for me, your call, but how can I emit the files on the npmFileMap parameter?

Something like ?:
      "files": [
        "**/*.+(js|css)",
        "!jquery-ui-timepicker-addon-i18n(.min)?.js"
      ]

**Comment by PeterDaveHello** [comment]
Nope, I didn't see it under `dist`:
```
jquery-ui-timepicker-addon/dist $ ls
i18n/       jquery-ui-sliderAccess.js       jquery-ui-timepicker-addon.js       jquery-ui-timepicker-addon.min.js
index.html  jquery-ui-timepicker-addon.css  jquery-ui-timepicker-addon.min.css
```

There is no need to touch pacakge.json again.

Just remove these files:
- ajax/libs/jquery-ui-timepicker-addon/1.6.1/jquery-ui-timepicker-addon-i18n.js
- ajax/libs/jquery-ui-timepicker-addon/1.6.1/jquery-ui-timepicker-addon-i18n.min.js

I am very happy to merge PR that's good too merge, if not, modifications will be needed, also spend my time, why do you think I'm kidding?

**Comment by ecabuk** [comment]
you are right, my fault. Duplicate files removed, thanks!
